src/javascripts/templates/templateSummary.js

```diff
@@ -0,0 +1,31 @@
+import { formatNumber } from "../helpers/format"
+
+class TemplateSummary {
+     /**
+     * Constructor of TemplateSummary object
+     */
+     constructor(){ }
+
+     /**
+     * HTML TemplateSummary for render an Summary object
+     * @param {object} summary 
+     */
+
```

Comment:
Remove this

Reply:
I fixed @phuocduong-agilityio . Please help me check. Thanks !